haven/code/game/objects/items/implants/implant_mindslave.dm

```diff
@@ -0,0 +1,75 @@
+/obj/item/implant/mindslave
+	name = "mindslave implant"
+	desc = "Turn a crewmate into your eternal slave"
+	activated = 0
+	implant_time = 300
+
+/obj/item/implant/mindslave/get_data()
+	var/dat = {"
+<b>Implant Specifications:</b><BR>
+<b>Name:</b> Syndicate Loyalty Implant<BR>
+<b>Life:</b> Single use<BR>
+<b>Important Notes:</b> <font color='red'>Illegal</font><BR>
+<HR>
+<b>Implant Details:</b> <BR>
+<b>Function:</b> Makes the injected a slave to the owner of the implant.<HR>"}
+	return dat
+
+/obj/item/implant/mindslave/implant(mob/source, var/mob/user)
+
+	if(!source.mind)
+		to_chat(user.mind, "<span class='notice'>[source] doesn't posses the mental capabilities to be a slave.</span>")
+		return 0
+
+	var/mob/living/carbon/human/target = source.mind.current
+	var/mob/living/carbon/human/holder = user.mind.current
+
+	if(target == holder)
+		to_chat(holder, "<span class='notice'>You can't implant yourself!</span>")
+		return 0
+
+	var/obj/item/implant/mindslave/imp = locate(src.type) in source
+	if(imp)
+		to_chat(holder, "<span class='warning'>[target] is already a slave!</span>")
+		return 0
+
+	if(target.isloyal())
+		to_chat(holder, "<span class='warning'>[target] seems to resist the implant!</span>")
+		return 0
+
+	to_chat(target, "<span class='userdanger'><FONT size = 3>You feel a strange urge to serve [holder.real_name]. A simple thought about disobeying his/her commands makes your head feel like it is going to explode. You feel like you dont want to know what will happen if you actually disobey your new master.</FONT></span>")
```

Comment:
Capitalize Master. Makes it flow better fluffwise.

Also make sure the text shows up irregardless of if the victim is conscious or not.

Reply:
yeah but it also makes it less gramatically correct, since you don't capitalize random nouns in english >:(

And I don't see why it wouldn't show up, since it's to_chat and not visible_message